src/consensus/aft/raft.h

```diff
@@ -2567,12 +2567,10 @@ namespace aft
       state->commit_idx = idx;
 
       LOG_DEBUG_FMT("Compacting...");
-      snapshotter->commit(idx);
-      if (consensus_type == ConsensusType::CFT)
-      {
-        // Snapshots are not yet supported with BFT
-        snapshotter->update(idx, replica_state == Leader);
-      }
+      // Snapshots are not yet supported with BFT
```

Comment:
While we don't support resuming from them in BFT, it seems to me that there should be no harm in emitting them?

Reply:
Yes, I was wondering that too and gave it a try but this currently doesn't work as the snapshot evidence generation logic doesn't go through the frontend like BFT would expect (instead, it creates a local `kv::Tx`, records the evidence and commits it "manually"). Is creating a new endpoint for this what we want? My assumption was that all nodes should generate the same snapshot in BFT, and record identical evidences in the store when they do so.

Reply:
If this involved creating an endpoint, let's not do it for now.